### DISCUSSION TITLE
Copy button for code in Tutorials

### DISCUSSION THREAD

**Discussion by jinalskothari** [kubernetes/website#43595] [Ideas]
I am doing the basic tutorial where I am expected to run commands such as `export POD_NAME="$(kubectl get pods -o go-template --template '{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}')"`. This kind of thing everybody would copy anyway.

I can imagine some arguments against this too. 
1. Potentially Dangerous
2. Typing commands helps in becoming proficient with the cli tools.

**Comment by divya-mohan0209** [comment]
Hello @jinalskothari, please [open an issue](https://github.com/kubernetes/website/issues/new?assignees=&labels=kind%2Ffeature&projects=&template=feature-request.md) against the kubernetes/website instead of starting a discussion item.
